src/akamai/js/akamaiHD_flash.js

```diff
@@ -0,0 +1,1334 @@
+/*
+ * Akamai HD flash video plugin
+ */
+
+require("../../../html5-common/js/utils/InitModules/InitOO.js");
+require("../../../html5-common/js/utils/InitModules/InitOOUnderscore.js");
+require("../../../html5-common/js/utils/InitModules/InitOOHazmat.js");
+require("../../../html5-common/js/utils/constants.js");
+(function(_, $) {
+  var pluginName = "ooyalaFlashVideoTech";
+  var flashMinimumVersion = "11.1.0";
+  var cssFromContainer;
+
+  /**
+   * Config variables for paths to flash resources.
+   */
+
+  var pluginPath;
+  var filename = "akamaiHD_flash.*\.js";
+  var scripts = document.getElementsByTagName('script');
+  for (var index in scripts) {
+    var match = scripts[index].src.match(filename);
+    if (match && match.length > 0) {
+      pluginPath = match.input.match(/.*\//)[0];
+      break;
+    }
+  }
+  if (!pluginPath) {
+    console.error("Can't get path to script", filename);
+    return;
+  }
+  pluginPath += "akamaiHD_flash.swf";
+  var flexPath = "playerProductInstall.swf";
+  this.ready = false;
```

Comment:
I believe that was in the original template.js we started from. It looks like it is gone from there now.
I see it in a december version of template.js. Not sure when it was removed. But it looks like we don't need it in the in PBF files either. We'll also test removing it from OSMF.